src/Dynatrace.OpenTelemetry.Exporter.Metrics.Tests/OneAgentMetadataEnricherTests.cs

```diff
@@ -63,12 +63,15 @@ public void IndirectionFileMissing()
         {
             var fileReader = Mock.Of<IFileReader>();
             Mock.Get(fileReader).Setup(f => f.ReadAllText(It.IsAny<string>())).Throws<FileNotFoundException>();
-            var targetList = new List<KeyValuePair<string, string>>();
+
+            var kv = KeyValuePair.Create<string, string>("defaultKey", "defaultValue");
```

Comment:
Maybe _initial_ would be better to avoid confusion with the "default dimensions" feature.

Reply:
renamed it to `initialDimensionList`